doc/libpmemobj/oid_is_null.3.md

```diff
@@ -116,6 +118,21 @@ The **OID_IS_NULL**() macro checks if *PMEMoid* represents a NULL object.
 
 The **OID_EQUALS**() macro compares two *PMEMoid* objects.
 
+For special cases where volatile (transient) variables need to be stored on
+persistent memory, there's a mechanism composed of *struct pmemvlt* type and
+**pmemobj_direct_volatile()** function. To use it, the *struct pmemvlt* needs to
+be placed at the beginning of the region of transient data. The *PMEMvlt* macro
+can be used to construct such a region.
+When the **pmemobj_direct_volatile()** function is called on a *struct pmemvlt*,
+it will return the pointer to the data and it will ensure that the provided
+constructor function is called exactly once in the current instance of the
+application.
```

Comment:
That's not completely true. The constructor may be called more than once when application opens and closes the same pool multiple times in one instance.

Reply:
done